cmd/neofs-cli/modules/acl/basic/print.go

```diff
@@ -0,0 +1,28 @@
+package basic
+
+import (
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/util"
+	"github.com/nspcc-dev/neofs-sdk-go/container/acl"
+	"github.com/spf13/cobra"
+)
+
+var printACLCmd = &cobra.Command{
+	Use:     "print",
+	Short:   "Pretty print basic ACL from the HEX representation",
+	Example: `neofs-cli acl basic print 0x1C8C8CCC`,
+	Long: `Pretty print basic ACL from the HEX representation.
+Few roles have exclusive default access to set of operation, even if particular bit deny it.
+Members with Container role have access to the operations of the data replication mechanism:
```

Comment:
We have `Container` role here, but not in the output. Can we omit the word `role` here?

Reply:
Changed.